recipes/joltphysics/all/test_package/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 3.8)
 project(test_package LANGUAGES CXX)
 
+add_definitions(-DJPH_PROFILE_ENABLED=0 -DJPH_DEBUG_RENDERER=0)
```

Comment:
```suggestion
```

The current recipe is exposing those definitions already: https://github.com/conan-io/conan-center-index/blob/master/recipes/joltphysics/all/conanfile.py#L162

Please, revert this change or explain what is failing.

Reply:
I kept getting an error after trying to run the executable saying that the library had been built with different definitions and so I hardcoded those in to fix it, I'll remove this so it can be done properly